apps/Vivaldi/install

```diff
@@ -2,7 +2,8 @@
 #https://help.vivaldi.com/desktop/install-update/manual-setup-vivaldi-linux-repositories/
 wget -qO- https://repo.vivaldi.com/archive/linux_signing_key.pub | gpg --dearmor | sudo tee /usr/share/keyrings/vivaldi-browser.gpg >/dev/null || exit 1
 echo "deb [signed-by=/usr/share/keyrings/vivaldi-browser.gpg arch=$(dpkg --print-architecture)] https://repo.vivaldi.com/archive/deb/ stable main" | sudo tee /etc/apt/sources.list.d/vivaldi-archive.list >/dev/null
-
+wget -O- https://launchpadlibrarian.net/574476800/chromium-codecs-ffmpeg-extra_97.0.4692.56-0ubuntu0.18.04.1_armhf.deb | tail -c+1153 | tar JOx ./usr/lib/chromium-browser/libffmpeg.so > libffmpeg.so.5.0
```

Comment:
I've never seen this approach before. Downloading a deb package, cutting it, and extracting a file from it, all simultaneously fascinates me.
How did you find this, and how does it work?

Reply:
I'm not sure I like the `tail` part much. it assumes the needed file starts at the 1153 byte and ends at the end of the file, and that might change. I think that a better more future proof way to do it would be to extract the deb and copy the file from there.

Reply:
So can you do it??